azurerm/internal/services/mssql/mssql_server_resource.go

```diff
@@ -150,6 +150,14 @@ func resourceArmMsSqlServer() *schema.Resource {
 				Computed: true,
 			},
 
+			"restore_dropped_database_id": {
```

Comment:
I realise there's already been some discussion on this, but I think the original `restorable_dropped_database_ids` is possibly more appropriate here as this computed field exposes those databases that have been dropped and are capable of being restored?

Reply:
I've changed the   `recover_database_id` to `recoverable_database_id`, too. As here, we could only apply one db, I've removed the plurality。